### PR TITLE
Add blend-bio

### PR DESCRIPTION
**Pull request by canfirtina**
Adding BLEND to bioconda.

Original GitHub page: https://github.com/CMU-SAFARI/BLEND
The arXiv link describing the paper: https://arxiv.org/abs/2112.08687

We will use the `blend-bio` package name because `blend` already exists and we would like to keep our tool name, `BLEND`, as is.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by canfirtina** [comment]
@BiocondaBot please add label

File: recipes/blend-bio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: blend-bio
+  version: 1.0.0
```
Comment (Redmar-van-den-Berg) [inline recipes/blend-bio/meta.yaml:5]: Can you use {{ version }} here as well, to make it easier to update the recipe in the future?

**Review by Redmar-van-den-Berg — CHANGES_REQUESTED** [review]

**Review by Redmar-van-den-Berg — APPROVED** [review]
Thanks!

**Comment by Redmar-van-den-Berg** [comment]
@BiocondaBot please merge

**Comment by canfirtina** [comment]
Thanks @Redmar-van-den-Berg !

**Comment by canfirtina** [comment]
@Redmar-van-den-Berg just double-checking if something went wrong with the merge. The latest build looks okay but the bot seems to complain about the merge in the last message.

**Comment by dpryan79** [comment]
@BiocondaBot please merge and stop being annoying.

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts (because if you give me another error I'll start getting annoyed with you).

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**